Formula/graphite2.rb

```diff
@@ -28,7 +41,7 @@ def install
   test do
     resource("testfont").stage do
       shape = shell_output("#{bin}/gr2fonttest Simple-Graphite-Font.ttf 'abcde'")
-      assert_match /67.*36.*37.*38.*71/m, shape
+      assert_match(/67.*36.*37.*38.*71/m, shape)
```

Comment:
I was about to blame Rubocop but double checking it's actually Ruby getting on my case about it. `ruby -wc` throws:

```
graphite2.rb:31: warning: ambiguous first argument; put parentheses or a space even after `/' operator
Syntax OK
```

Can always revert if you prefer.